npm-debug.log

```diff
@@ -0,0 +1,47 @@
+0 info it worked if it ends with ok
```

Comment:
😞  ... remove this file for now. I added `*.log` to the .gitignore in master for future